xbmc/FileItem.cpp

```diff
@@ -2854,6 +2854,9 @@ void CFileItemList::RemoveExtensions()
     m_items[i]->RemoveExtension();
 }
 
+static constexpr int FOLDER_CANDIDATE = 1;
+static constexpr int FILE_CANDIDATE = 2;
+
 void CFileItemList::Stack(bool stackFiles /* = true */)
```

Comment:
The parameter is not used anymore it seams and thus can be removed.

Reply:
Done

Reply:
Done